### ISSUE TITLE
Issue when used to install nix on Mac OS Ventura 13.1

### ISSUE THREAD

**Issue by rgasper** [jetpack-io/devbox#400]
I see the following in the logs when installing Nix. Not entirely sure this is going to be an issue in using devbox, but there's definitely a bug:

```
~~> Setting up the basic directory structure
chown: /nix/.Trashes: Operation not permitted
chown: /nix/.Trashes: Operation not permitted
```

**Comment by loreto** [comment]
Thanks for reporting! We'll take a look. It might take us a bit longer than usual as we're entering the holiday season and some of the team is away; but we'll report back on this ticket once we address.

**Comment by mikeland86** [comment]
Thanks for the report! Two quick questions:

1. Is this your first nix install? Or is this a reinstall? (This error can happen when you have an existing `/nix` directory before the install)
2. If you try to install nix directly (https://nixos.org/download.html#nix-install-macos) do you get same issue?

It is possible this doesn't affect devbox. If that's the case let us know as well!

Thanks!

**Comment by Lagoja** [comment]
I tested this on Mac OS Ventura 13.1 with Devbox 0.2.0. I did get the same error in my logs, but the nix install seemed to proceed as normal, and I got a successful Nix installation that worked with Devbox. So right now this doesn't seem to affect Devbox

I think this may happen when a `/nix` store already exists on the machine. If it does end up breaking the install, the correct next steps would be to [uninstall Nix completely](https://nixos.org/manual/nix/stable/installation/installing-binary.html#uninstalling) and try again.
